optionals/diagnostic_disable_xeh_logging/config.cpp

```diff
@@ -11,6 +11,9 @@
 
 class CfgPatches {
     class Disable_XEH_Logging {
+        author = "$STR_CBA_Author";
+        name = "$STR_CBA_Optional_Component";
```

Comment:
This addon is very simple and does not `#include...` the usual bits and pieces, hence the use of a literal stringtable name.